rspack.config.js

```diff
@@ -377,4 +383,20 @@ if (devMode) {
   );
 }
 
-module.exports = config;
+const additionalRspackConfig = [];
+
+if (process.env.MB_EDITION === "ee") {
+  const SDK_IFRAME_EMBED_PLUGIN_PATH = `${ENTERPRISE_SRC_PATH}/embedding_iframe_sdk`;
```

Comment:
I'd consult some license experts here. Normally, we put stuff plugins under `metabase-enterprise` and we assign properties to plugins there. e.g. `PLUGIN_WHATEVER.property = xxx`

then we use this plugin in OSS code just fine.

For this case, we're technically importing EE code in an OSS file (it's not under enterprise directory), and we don't allow that on FE, so you can't import `EE` code inside OSS code (no `import xxx from 'metabase-enterprise/*` when we're inside `frontend/src/metabase/`.

Can we stick to the current approach? You can put `embedding_iframe_sdk` inside OSS file alongside `app-embed.js` but the components it renders are plugins.

Reply:
That makes sense, let me move the entry point to live with its `app-*.js` friends then!

Reply:
Fixed in https://github.com/metabase/metabase/pull/56915/commits/bd189b7862dbba16903cc67643712c086603b74c, moved to use EE plugins!

Alberto wants folks to be able to try out the iframe embedding on EE builds without a valid license, so I always assign the EmbedRoute component on the EE plugin. I then added a separate `hasValidLicense` field that is guarded behind `hasPremiumLicense(token)` that we can use.